tests/main/refresh-all-undo/task.yaml

```diff
@@ -23,6 +23,17 @@ prepare: |
     echo "And the daemon is configured to point to the fake store"
     setup_fake_store $BLOB_DIR
```

Comment:
Yay, thanks for finding the root cause!

Would it make sense to make this wait part of "setup_fake_store()"? This way we don't need to remember to add the wait code when using setup_fake_store